model/flow/service_event_test.go

```diff
@@ -103,11 +121,67 @@ func TestEncodeDecode(t *testing.T) {
 			require.True(t, ok)
 			assert.DeepEqual(t, setup, gotSetup)
 
+			t.Logf("- debug: setup.ServiceEvent()=%+v\n", setup.ServiceEvent())
 			b, err = msgpack.Marshal(commit.ServiceEvent())
 			require.Nil(t, err)
 
 			outer = new(flow.ServiceEvent)
+			t.Logf("- debug: outer=%+v <-- before .Unmarshal()\n", outer)
 			err = msgpack.Unmarshal(b, outer)
+			t.Logf("- debug: outer=%+v <-- after .Unmarshal()\n", outer)
+			require.Nil(t, err)
```

Comment:
@yhassanzadeh13 thanks for the comment!

Yep, I just copy and pasted the code already there for the existing tests for the other serialization formats. Now, I mercilessly refactored the whole file to use `require.NoError()` instead of `require.Nil()` :-)